{"topic": "Build up radiance error", "category": "radiance-general", "attachments": [], "created_by_name": "\u51b7\u9762\u5bd2\u67aa", "created_at": "April 23, 2015 at 10:51PM", "body": "Hi,erveryone\n When I build up radiance using scons ,following errors are shown:\nTypeError: sequence item 0: expected string, function found:\n  File \"/home/zhaoyongqing/Radiance-5.0.a.1/SConstruct\", line 57:\n    load_plat.load_plat(env, ARGUMENTS, platform=None)\n  File \"/home/zhaoyongqing/Radiance-5.0.a.1/build_utils/load_plat.py\", line 78:\n    read_plat(env, args, pfn)\n  File \"/home/zhaoyongqing/Radiance-5.0.a.1/build_utils/load_plat.py\", line 12:\n    cfig.read(fn)\n  File \"/usr/lib/python2.7/ConfigParser.py\", line 305:\n    self._read(fp, filename)\n  File \"/usr/lib/python2.7/ConfigParser.py\", line 554:\n    options[name] = '\\n'.join(val)\n\n\nwhich is why?My radiance source package version is 5.0.a.1,and phython version is phython2.7.\n\n\n\n\n\n\n Yongqing Zhao\n Changsha Green Building & Energy Saving Technology CO.,LTD\nNO.438,Shaoshan Road,Changsha,Hunan,China\nTelephone:13574805636\n Email:zhaoyongqing1987 at 126.com\n          503271081 at qq.com\n___\n<sup>Automatically generated content from [radiance mailing-list](https://radiance-online.org/pipermail/radiance-general/2015-April/010853.html).</sup>", "id": "radiance-general_010853", "created_by": "\u51b7\u9762\u5bd2\u67aa"}